### ISSUE TITLE
github ???

### ISSUE THREAD

**Issue by AroKol78** [martok/palefill#97]
Serpent52(20240105025008)  + palefill1.27
Recently GH has messed up and the site has **significantly slowed down** which is beyond my tolerance for browsing.
I managed to capture the slowdown during which it displays strange things.
There would be a way around this?

<details><summary>Screenshot_1</summary>

`https://github.com/uBlockOrigin/uAssets/commits/master/`
![ghbug0001](https://github.com/martok/palefill/assets/70756447/66e059cc-0074-4ab5-9b41-34dc6699e47b)

</details>

<details><summary>Screenshot_2</summary>

`https://github.com/uBlockOrigin/uAssets`
![ghbug0002](https://github.com/martok/palefill/assets/70756447/5b053c1e-5072-46f7-b18d-617c6b625c10)

</details>

**Comment by RamonUnch** [comment]
You may try to disable palefill on github, it is no longer required, However I am not sure anything can be done, even when disabling palefill GitHub is very slow for me as well and I have to restart the browser frequently, because of memory leaks.

If you disable palefill you may need to manually set `dom.enable_performance_observer;true` in `about:config`.

**Comment by martok** [comment]
Fascinating, the rendering glitch while loading aside, the new (React, I believe) version is significantly faster for me.

> and I have to restart the browser frequently, because of memory leaks.

That, on the other hand, has always been the case for me with Palemoon. Once the allocated memory exceeds ~1.5 GB, I just have to restart it. If I don't, at about 2G GC times just make it unresponsive.

**Comment by Vangelis66** [comment]
> If you disable `palefill`, you may need to manually set `dom.enable_performance_observer;true` in **about:config**.

... Unlike official **Pale Moon** (and **NM28**), official **Basilisk** as well as **Serpent 52** (OP's case) now have that pref default to `true` (i.e., there's no need to mess with it anymore in said browsers) 😉 ...

**Comment by jrfnl** [comment]
I've been seeing the same rendering artifacts for quite a while now and find that the page lading is excruciatingly slow for things like the Milestone/Label dropdowns in issues/PRs.

As for the restarting, I do need to do this regularly as well, but generally only once it reached > 5G memory used. Might this be dependent on the total available memory ? (I have 64G available total).

**Comment by martok** [comment]
> As for the restarting, I do need to do this regularly as well, but generally only once it reached > 5G memory used. Might this be dependent on the total available memory ? (I have 64G available total).

Huh, that's odd. 16GB total here, so 1/4. 5G/4 = 1.2G, thats awefully close for a coincidence. OTOH, my 8 GB laptop is still fine at 1GB for PM. The configurable thresholds are all absolute values, but maybe something internal isn't...

**Comment by AroKol78** [comment]
it started pouring out errors

```
Error: SyntaxError: ':modal' is not a valid selector
Source File: https://github.githubassets.com/assets/vendors-node_modules_oddbird_popover-polyfill_dist_popover_js-7bd350d761f4.js
Line: 1
```
and
```
Warning: Ignoring unsupported entryTypes: event.
Source File: https://github.githubassets.com/assets/behaviors-91d3668ba8db.js
Line: 33
```

**Comment by AroKol78** [comment]
again :(

1. you can wait forever, the worst thing is that you can't see longer `issues`
2. the link does not open, you have to use it in a new tab to open it

![Image](https://github.com/user-attachments/assets/b2480214-f98a-4e91-a548-16fe06ab67c2)

**Comment by jrfnl** [comment]
@AroKol78 GitHub is working on a new issue view and it's horrible and doesn't work _at all_ in Palemoon.
More info here: https://github.blog/changelog/2025-01-13-evolving-github-issues-public-preview

**Comment by SlySven** [comment]
I take it that if we moan at GH (https://github.com/orgs/community/discussions/148713) they'll say "Oh you should be using a modern supported browser."? Expecting everyone to swallow the Chromium *cool-aide*. :rage:

**Comment by jrfnl** [comment]
@SlySven @AroKol78 For the time being (once the roll-out has reached your account and you're not just seeing it because some organisation has opted-in), you can disable it from your account's "feature preview" settings:
![image](https://github.com/user-attachments/assets/9e367c4b-945b-4776-8e79-74b862b3be34)

Not sure how long that will last, but it helps for now.

**Comment by Vangelis66** [comment]
> you can disable it

... Most sadly, however, this `New Issues Experience` **abomination** :rage: has been forced upon **non-logged-in** GitHub users _without the option to disable_ the "Preview" :disappointed: ; so, if you're not already a GH member and want to read an open-source project's Issue Tracker, you're already screwed...

Microsoft employees :-1: seem to code only having **latest ChrEdge** (under **Win11**) as target client; every redesign of theirs is to the detriment of "older" browser engines (suitable for "older"/low-resourced H/W), **UXP** platform always paying a toll as a result of their "overhauls"; truth be told, **GitHub** has been in several ways broken under **UXP** (due to lingering CSS issues), but if you were prepared to turn a blind eye to those "cosmetic" annoyances, GH would still be function-able...

And the **UXP** team have probably thrown in the towel when it comes to these **GitHub web-compat issues**; I'm not optimistic myself this (and further) GH breakage will be properly addressed by them; the standard answer on the PM forums seems to be now "Use Firefox/Chromium" (sadly, the users of **UXP forks** on XP/Vista[/7] don't have that option; apart from, maybe, Supermium, but this is OT) ...

> Not sure how long that will last

... Based on their previous "experiments" (e.g. the "**Global Navigation Update**"), it'll take from 3 to 6 months _top_ until the "feature" becomes the norm; already foisted upon anonymous users, as I wrote above, so am leaning towards 3 months myself...

**Comment by jrfnl** [comment]
@Vangelis66 Totally with you on the frustration. The diversity of the web is being broken and actively worked against by these type of changes. Forcing everyone onto (horrible, bad for privacy, no usability for large number of tabs etc etc) chromium-based browsers is a form of killing browser competition on the web and I'm just waiting for an anti-trust legal case.

**Comment by Vangelis66** [comment]
... This has now made it to the **official PM Forums**: 

https://forum.palemoon.org/viewtopic.php?f=70&t=32021

It appears to be caused by code **UXP doesn't support**: 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

**Chrome** supported this early on (since v**37**), but **Fx** only relatively recently (since v**98**); **Fx**'s implementation is, for sure, non "portable" as-is across to **UXP**...

**Comment by martok** [comment]
> It appears to be caused by code **UXP doesn't support**:
> 
> https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

Yes it does. The selector is fixed upstream.
